_posts/2024-12-05-release-notes-3.6.2.md

```diff
@@ -217,16 +179,76 @@ Let's take the following example:
 ```
 
 Reordering the fields is binary-compatible but it might affect the meaning of `@Annotation(1)`
-Starting from Scala 3.6.0, named arguments are required for Java-defined annotations.
+Starting from Scala 3.6, named arguments are required for Java-defined annotations.
```

Comment:
There is one exception to this, which I think we should mention (possibly with an example).
It's when only a single argument is passed to an annotation and the annotation defines a parameter called `value`.

Reply:
I've added a section for that with an example